src/ToolBox/superpmi/superpmi-shared/icorjitinfoimpl.h

```diff
@@ -125,6 +125,16 @@
             unsigned*                   offsetAfterIndirection  /* OUT */
             );
 
+    CORINFO_METHOD_HANDLE resolveVirtualMethod(
+        CORINFO_METHOD_HANDLE virtualMethod,
+        CORINFO_CLASS_HANDLE implementingClass
+        );
+
+    CORINFO_METHOD_HANDLE resolveVirtualMethodExact(
```

Comment:
It would be useful to have some comments what these methods do. (In particular, what's the difference between them.)